app/models/import/actual_history.rb

```diff
@@ -21,6 +21,8 @@ def call
     import_actual_history(@csv)
   end
 
+  alias imported? call
+
   private
```

Comment:
I think this is a really interesting topic. At this month's LRUG meeting, there was a thought-provoking presentation about how we are too often reach for this "service object" or "command" design pattern (https://github.com/davidgf/design-patterns-in-ruby/blob/master/command.md) -- where we might more profitably think about "domain objects". I think this critique resonated with a lot of the group.

15 min video if you want to watch https://assets.lrug.org/videos/2021/november/patricia-cupueran-service-objects-and-domain-objects-differences-lrug-nov-2021.mp4

Reply:
Bah! I was supposed to go but family stuff came up! I definately feel like on this project we have lost sight (or perhaps never really had) what a 'service object' is over a regular 'object' its on my list of things to dig into so I have an opinion in the future - be good to get your thoughts as well.  

Thanks for the link! I'll add it to my list of research!